linodecli/helpers.py

```diff
@@ -120,3 +123,35 @@ def expand_globs(pattern: str):
         print(f"No file found matching pattern {pattern}")
 
     return [Path(x).resolve() for x in results]
+
+
+T = TypeVar("T")
+
+
+def sorted_actions_smart(
```

Comment:
This doesn't live with the rest of the util functions because it can also be used in the command --help pages